types/src/traits/election.rs

```diff
@@ -235,8 +243,13 @@ pub trait ConsensusExchange<TYPES: NodeType, LEAF: LeafType<NodeType = TYPES>, M
     }
 
     /// Threshold required to approve a [`Proposal`](Self::Proposal).
-    fn threshold(&self) -> NonZeroU64 {
-        self.membership().threshold()
+    fn success_threshold(&self) -> NonZeroU64 {
```

Comment:
This is an interesting change. Does that mean we assume all exchanges support yes/no votes?

Reply:
I figured we'd want to in the future, like how we want to allow Yes No votes for DA.  What do you think?

Reply:
I believe we have cases where we won't want to use No votes - including the DA certificate - but a) I don't think it does any harm, and b) in the best case, it'll allow speeding up the next round of DA committee size expansions.